server/src/main/java/org/jboss/as/server/mgmt/UndertowHttpManagementService.java

```diff
@@ -73,7 +73,9 @@
 public class UndertowHttpManagementService implements Service<HttpManagement> {
 
     public static final RuntimeCapability<Void> EXTENSIBLE_HTTP_MANAGEMENT_CAPABILITY =
-            RuntimeCapability.Builder.of("org.wildfly.management.http.extensible", ExtensibleHttpManagement.class).build();
+            RuntimeCapability.Builder.of("org.wildfly.management.http.extensible", ExtensibleHttpManagement.class)
+                    .addAdditionalRequiredPackages("org.jboss.as.domain-http-error-content")
```

Comment:
A remark that is not related to your PR but I get confused in all the different "names" we have (module name, package name, feature name) that all uses String.

Maybe we could add something similar to `ServiceName` to type our API.